app/views/users/new.html.haml

```diff
@@ -9,6 +15,30 @@
           = f.password_field :password
           = f.text_field     :fullname, label: 'Full Name'
           = hidden_field_tag :invitation_token, params[:token]
+        %fieldset.credit_card
+          %span.payment-errors
+          .form-group
+            = label_tag nil, "Credit Card Number", class: "control-label col-sm-2"
+            .col-sm-6
+              = text_field_tag nil, nil, class: "form-control", data: {stripe: "number"}
+          .form-group
+            = label_tag nil, "Security Code", class: "control-label col-sm-2"
+            .col-sm-6
+              = text_field_tag nil, nil, class: "form-control", data: {stripe: "cvc"}
```

Comment:
Lines 21 to 27 are a bit wrong - you should not use Rails Helpers - its not benefiting it as you don't want to pass this data to Controller. Sure there is a way around it using these helpers but they are in my opinion more verbose solution, misleading a bit. Why would you create a 'text_field_tag nil ...` I understand your idea - not passing name attr but in this case using pure HTML is more preferred. To make Capybara work with these tags, you also want to make for attribute active for label and have these text field tags an ID. Strongly recommend to fix this based on solution from video.

Hope this helps :)